src/controller/AdminController.php

```diff
@@ -242,9 +270,13 @@ private function controleForms(UserModel $userModel, PostModel $postModel, Comme
 	 * render Template.
 	 *
 	 * @param string $templateName Template name to render
-	 * @param array $error error information to display
-	 *
 	 * @param array $session user session
+	 * @param array $error error information to display
+	 * @param object $pendingUsers return pending user
```

Comment:
```suggestion
	 * @param array $pendingUsers return pending user
```

idem pour les suivants

Reply:
Oui, sans le return, cela affiche le dashbord à la suite de l'erreur